### PR TITLE
add start-index and stop-index property in literal-expression for PR#5006

### LINKED ISSUE #4995: Add parsing integrated test for `update` query SQL
**Content**: Add two parsing integrated tests for `update` query SQL.
**Difficulty rating**: ★☆☆☆☆

### How to do
1. Add the following SQL cases to `/Users/trista/Documents/sharding-code-trista/incubator-shardingsphere/sharding-sql-test/src/main/resources/sql/supported/dml/update.xml`

**SQL case ID**: update_with_set_calculation
> UPDATE t_order SET status = status - ? WHERE order_id = ? AND user_id = ?
parameters: 1, 2, 3

**SQL case ID**: update_with_where_calculation
> UPDATE t_order SET status = ? WHERE order_id = order_id - ? AND user_id = ?
parameter: 1, 2, 3

2. Calculate the parsing results of the above SQL cases and add those parsing results to `./incubator-shardingsphere/shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/resources/sql/dml/update.xml`
**Tip**: You can refer to the parsing result of SQL case ID `update_without_alias` in the same directory.

3. Run `SQLParserParameterizedTest` to test your SQL cases.

### PR DESCRIPTION
**Pull request by strongduanmu**
Fixes #4995 .

Changes proposed in this pull request:
- add start-index and stop-index property in literal-expression for PR#5006
-
-

### CODE REVIEW

File: shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/resources/sql/dml/update.xml [hunk 1]
```diff
@@ -386,8 +386,8 @@
             <assignment start-index="19" stop-index="28" literal-stop-index="28">
                 <column name="status" start-index="19" stop-index="24" />
                 <assignment-value>
-                    <parameter-marker-expression value="0" start-index="28" stop-index="28" />
```
Comment (tristaZero) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/resources/sql/dml/update.xml]: Hi Please keep this blank space.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/resources/sql/dml/update.xml [hunk 2]
```diff
@@ -404,8 +404,8 @@
                     <column-left-value name="user_id" start-index="64" stop-index="70" literal-start-index="64" literal-stop-index="70" />
                     <operator type="=" />
                     <compare-right-value>
-                        <parameter-marker-expression value="2" start-index="74" stop-index="74" />
```
Comment (tristaZero) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/resources/sql/dml/update.xml]: Hi Please keep this blank space.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/resources/sql/dml/update.xml (hunk 1 above)
Comment (strongduanmu) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/resources/sql/dml/update.xml]: > Hi Please keep this blank space.

Ok, I'll modify it.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/resources/sql/dml/update.xml (hunk 2 above)
Comment (strongduanmu) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/resources/sql/dml/update.xml]: > Hi Please keep this blank space.

Ok, I'll modify it.

**Comment by tristaZero** [comment]
Appreciated your carefulness. :)

**Pull request merged**